sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractEditForm.java

```diff
@@ -79,7 +79,8 @@ protected AbstractEditForm(
 		this.fieldAccessCheckers = fieldAccessCheckers;
 
 		getFieldGroup().addCommitHandler(this);
-		setWidth(900, Unit.PIXELS);
+		setWidth(1100, Unit.PIXELS);
```

Comment:
Revert this change because it might have unintended side effects (the width is set to 900 for a reason). Instead, please move the buttons to clear the symptoms and set the cleared symptoms to no/unknown to a new line to make sure that they don't interfer with the description. They should still be right-aligned though.